README.md

```diff
@@ -50,42 +49,41 @@
 
```

Comment:
- [ ] It would be nice if you add screenshots of all the queries which you have made in the queries.sql file for this milestone so it would be easy to for the next reviewer to verify that your queries are giving expected results 👍